src/seedu/addressbook/Main.java

```diff
@@ -31,24 +32,31 @@
     private List<? extends ReadOnlyPerson> lastShownList = Collections.emptyList();
 
 
-    public static void main(String... launchArgs) {
+    public static void main(String... launchArgs) throws FileNotFoundException {
         new Main().run(launchArgs);
     }
 
-    /** Runs the program until termination.  */
+    /** Runs the program until termination.  
+     * @throws FileNotFoundException */
     public void run(String[] launchArgs) {
-        start(launchArgs);
-        runCommandLoopUntilExitCommand();
-        exit();
+    	try{
+            start(launchArgs);
+            runCommandLoopUntilExitCommand();
+            exit();
+    	}catch(FileNotFoundException fnfe){
```

Comment:
A good catch.

Probably need new error message though as the file missing operation can happen not only at the beginning (initialization) but throughout the execution period.